apps/login/src/app/(login)/idp/[provider]/success/page.tsx

```diff
@@ -201,7 +153,8 @@ export default async function Page(props: {
     }
   }
 
-  if (options?.isCreationAllowed && options.isAutoCreation) {
+  // if link === true, do not create user
+  if (options?.isCreationAllowed && options.isAutoCreation && !link) {
```

Comment:
What about this?

```suggestion
  if (link) {
    return linkingFailed(branding);
  }
  
  if (options?.isCreationAllowed && options.isAutoCreation) {
```

Reply:
BTW, don't we want to show a hint why linking failed?

Reply:
done